executor/distsql.go

```diff
@@ -618,6 +655,7 @@ func (e *IndexLookUpExecutor) buildTableReader(goCtx goctx.Context, handles []in
 		tableID:      e.tableID,
 		dagPB:        e.tableRequest,
 		priority:     e.priority,
+		feedback:     statistics.NewQueryFeedback(0, 0, 0, 0),
```

Comment:
it's should not be 4 `0`s ?

Reply:
It is meaningless for this table reader to have query feedback, so here just use four `0`s.